Description: Fix typo in binary
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Forwarded: yes
Applied-Upstream: https://github.com/poolpOrg/OpenSMTPD/commit/e39350501f7cbf5e393f73db800d05d5573e6204
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/smtpd/scheduler.c
===================================================================
--- opensmtpd.orig/smtpd/scheduler.c	2013-07-30 11:20:30.511092111 -0400
+++ opensmtpd/smtpd/scheduler.c	2013-07-30 11:20:30.511092111 -0400
@@ -101,7 +101,7 @@
 		m_get_msgid(&m, &msgid);
 		m_end(&m);
 		log_trace(TRACE_SCHEDULER,
-		    "scheduler: commiting msg:%08" PRIx32, msgid);
+		    "scheduler: committing msg:%08" PRIx32, msgid);
 		n = backend->commit(msgid);
 		stat_decrement("scheduler.envelope.incoming", n);
 		stat_increment("scheduler.envelope", n);
Index: opensmtpd/smtpd/mproc.c
===================================================================
--- opensmtpd.orig/smtpd/mproc.c	2013-06-06 09:28:22.391357396 -0400
+++ opensmtpd/smtpd/mproc.c	2013-07-30 11:20:58.499894096 -0400
@@ -675,7 +675,7 @@
 	m_get_typed_sized(m, M_ENVELOPE, &d, &s);
 
 	if (!envelope_load_buffer(evp, d, s - 1))
-		fatalx("failed to retreive envelope");
+		fatalx("failed to retrieve envelope");
 	evp->id = evpid;
 #endif
 }
